www/vpn-ban-notice/index.html

```diff
@@ -0,0 +1,87 @@
+    <link rel="icon" type="image/png" href="/favicon-196x196.png" sizes="196x196">
+    <link rel="icon" type="image/png" href="/favicon-160x160.png" sizes="160x160">
+    <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96">
+    <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
+    <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
+    <meta name="msapplication-TileColor" content="#2d89ef">
+    <meta name="msapplication-TileImage" content="/mstile-144x144.png">
+
+    <meta name="viewport" content="width=device-width">
+
+    <meta http-equiv="cache-control" content="max-age=0" />
+    <meta http-equiv="cache-control" content="no-cache, no-store, must-revalidate" />
+    <meta http-equiv="expires" content="0" />
+    <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
+    <meta http-equiv="pragma" content="no-cache" />
+
+    <title>Ban Notice - DDraceNetwork</title>
+  </head>
+  <body>
+    <article>
+      <header>
+        <div class="fade">
+          <menu class="contentleft">
+            <div class="title">
+              <h1><a href="/"><img class="logobig" alt="DDraceNetwork" src="/ddnet2.svg"/><img class="logosmall" alt="DDraceNetwork" src="/ddnet.svg"/></a></h1>
+            </div>
+            <ul class="big">
+              <li><a href="/status/">Status</a></li>
+              <li><a href="/ranks/">Ranks</a></li>
+              <li><a href="/releases/">Map Releases</a></li>
+              <li><a href="/discord">Discord</a> / <a href="//forum.ddnet.org/">Forum</a></li>
+              <li><a href="//wiki.ddnet.org/">Wiki</a></li>
+              <li><a href="/downloads/">Downloads</a></li>
+              <li><a href="/tournaments/">Tournaments</a></li>
+              <li><a href="/skins/">Skin Database</a></li>
+              <li><a href="/stats/">Statistics</a></li>
+              <li><a href="/staff/">Staff &amp; Contact</a></li>
+              <li><a href="/switch-theme/">Switch Theme</a></li>
+            </ul>
+            <ul>
+              <li><a href="/#news">News</a> (<a href="/news/">all</a>)</li>
+              <li><a href="/server/">Server&nbsp;Features</a></li>
+              <li><a href="/client/">Client&nbsp;Features</a></li>
+              <li><a href="/map/">Map&nbsp;Features</a></li>
+              <li><a href="/howto/">How&nbsp;to&nbsp;Map</a></li>
+              <li><a href="/rules/">Mapping&nbsp;Rules</a> / <a href="/guidelines/">Guidelines</a></li>
+              <li><a href="/binds/">Useful&nbsp;Binds</a></li>
+              <li><a href="/auto-font/">Auto-Font</a></li>
+              <li><a href="/settingscommands/">Settings&nbsp;&amp;&nbsp;Commands</a></li>
+              <li><a href="/renames/">Renames</a></li>
+              <li><a href="/funding/">Funding</a></li>
+            </ul>
+          </menu>
+        </div>
+      </header>
+      <section>
```

Comment:
This should all be replaced by
```suggestion
---
layout: default
title: VPN Ban Notice - DDraceNetwork
menu: |
  <ul>
    <li><a href="/#news">News</a> (<a href="/news/">all</a>)</li>
    <li><a href="/server/">Server&nbsp;Features</a></li>
    <li><a href="/client/">Client&nbsp;Features</a></li>
    <li><a href="/map/">Map&nbsp;Features</a></li>
    <li><a href="/howto/">How&nbsp;to&nbsp;Map</a></li>
    <li><a href="/rules/">Mapping&nbsp;Rules</a> / <a href="/guidelines/">Guidelines</a></li>
    <li><a href="/binds/">Useful&nbsp;Binds</a></li>
    <li><a href="/auto-font/">Auto-Font</a></li>
    <li><a href="/settingscommands/">Settings&nbsp;&amp;&nbsp;Commands</a></li>
    <li><a href="/renames/">Renames</a></li>
    <li><a href="/funding/">Funding</a></li>
  </ul>
---
```